sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -53,21 +56,23 @@ internal class OperationInternal<T> : OperationInternalBase
     {
         private readonly IOperation<T> _operation;
         private readonly AsyncLockWithValue<OperationState<T>> _stateLock;
-        private Response _rawResponse;
+        private Response? _rawResponse;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="OperationInternal"/> class in a final successful state.
         /// </summary>
         /// <param name="rawResponse">The final value of <see cref="OperationInternalBase.RawResponse"/>.</param>
         /// <param name="value">The final result of the long-running operation.</param>
-        public static OperationInternal<T> Succeeded(Response rawResponse, T value) => new(OperationState<T>.Success(rawResponse, value));
+        /// <param name="rehydrationToken">rehydration token</param>
+        public static OperationInternal<T> Succeeded(Response rawResponse, T value, RehydrationToken? rehydrationToken = null) => new(OperationState<T>.Success(rawResponse, value), rehydrationToken);
```

Comment:
Where are the tests for the new methods?

Reply:
This is not a new method, added an optional input parameter to be used for fake LRO scenario.